connector/src/it/scala/com/datastax/spark/connector/datasource/CassandraCatalogNamespaceSpec.scala

```diff
@@ -7,7 +7,7 @@ import scala.collection.JavaConverters._
 class CassandraCatalogNamespaceSpec extends CassandraCatalogSpecBase {
 
   "A Cassandra Catalog Namespace Support" should "initialize successfully" in {
-    spark.sessionState.catalogManager.currentCatalog.name() should include("Catalog cassandra")
+    spark.sessionState.catalogManager.currentCatalog.name() should be(catalogName)
```

Comment:
I may or may not have ... added this test in a ton of other catalog specs

[info]   "cassandra" did not include substring "Catalog cassandra" (CassandraCatalogTableReadSpec.scala:22)
2292
You'll probably need to hit them all

Reply:
Other than that, Patch lgtm :)

Reply:
indeed! thanks, fixed.